plugins/publish_score.js

```diff
@@ -0,0 +1,30 @@
+const secret = "verysecret";
+const drogue_url = "https://http.sandbox.drogue.cloud/v1/scores";
+const application = "jsgame";
+const device = "client";
+
+const auth = btoa(device+"@"+application+":"+secret)
```

Comment:
the device needs to URL encoded first

Reply:
why? It's not used in the URL

Reply:
because the device may have special characters, like `@` or `:` or :hankey: or :robot: … and those need to be encoded, so that the still is `<device>@<application>` … so the `<device>` part must be urlencoded. Application not, as it is supposed to be a hostname.